source/index.rst

```diff
@@ -51,6 +51,7 @@ These are institutions who were early adopters or provided HPC resources for dev
    :caption: General
 
    architecture
+   request-flow-diagrams
```

Comment:
I want to be cautious in adding new headings here. Indeed I'll be making pull request to reduce them.

It's fine to be it's own page, but would this make more sense as a subheading within architecture (or some other heading)?